api/src/main/java/io/druid/data/input/impl/TimeAndDimsParseSpec.java

```diff
@@ -64,6 +64,12 @@ public void setFieldNames(Iterable<String> fieldNames)
       {
         throw new UnsupportedOperationException("not supported");
       }
+
+      @Override
+      public void reset()
```

Comment:
Shouldn't be necessary if you make `reset` a default method (similar comment for other impls)